test/plugins/features/movies/index.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const Movies = require('../../../../lib/server');
+
+describe('movies integration', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
+      return Movies.inject({
```

Comment:
If this test is being added, can it be made a/a?

Reply:
i think i added that in earlier but forgot to give it its own commit, so it got lumped in with this commit. I plan on making the changes for test files after the controller gets approval

Reply:
Ok, but the functionality is going into this commit. 

I guess it's fine for now.